src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 0;
+
+  if (days < 3) {
+    cost = days * 40;
+  } else if (days >= 3 && days < 7) {
+    cost = (days * 40) - 20;
+  } else if (days >= 7) {
+    cost = (days * 40) - 50;
+  }
+
+  return cost;
```

Comment:
```suggestion
  if (days < 3) {
   return days * 40;
  } 
  
  if (days >= 3 && days < 7) {
    return  (days * 40) - 20;
  } 
  
  return cost = (days * 40) - 50;
```
try to avoid else if where it is possible